tests/libtest/test_pod_exec.py

```diff
@@ -1,9 +1,12 @@
 import os
+import logging
 os.sys.path.append(os.path.dirname(os.getcwd()))
 
 from ocs_ci.framework.testlib import libtest
 from ocs_ci.ocs.resources import pod
 
+logger = logging.getLogger(__name__)
+
 
 @libtest
 def test_main():
```

Comment:
I would rename this to `test_pod_exec`. I know that the module is named that but `test_main` is too generic.